__tests__/app.test.js

```diff
@@ -48,3 +48,36 @@ describe("/api", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+    test("should respond with an article object for the specified article", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("body");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
```

Comment:
Great that we've ensured the properties exist - but we could make this an even stronger test by also checking the type of the value that is returned.

There's a few ways we can do this - toHaveProperty can actually accept a second argument for the expected value like so:
          expect(article).toHaveProperty("author", expect.any(String));
          
OR, you can use the jest matcher .toMatchObject for a similar purpose:

expect(article).toMatchObject({
author : expect.any(String)
<... other properties>
})